client/src/components/ExpressionResult.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="exp-box">
+    <div class="left-exp-box"></div>
+    <div class="right-exp-box"></div>
+  </div>
+</template>
+
+<style scoped>
+.exp-box {
+  height: 24.875rem;
+  width: 102.6875rem;
+  display: flex;
+  background-color: rgba(0, 0, 0, 0.25);
+}
+.left-exp-box {
+  height: 24.875rem;
+  width: 46.5625rem;
```

Comment:
外のコンテナにサイズ指定があるのでこれでいいと思います
```suggestion
  flex: 1;
```